tcf_website/templates/about/sponsors.html

```diff
@@ -0,0 +1,20 @@
+{% block subcontent %}
+{% load static %}
+
+<div class="tCF-sponsors">
+  <div class="grid">
+    <h1 class="grid-header">Thanks to our Sponsors</h1>
+    <div class="sponsors-text">
+      <p>
+        Our sponsors help us cover server costs from the large amount of traffic that we get.
+        Their help has allowed the site to stay up and running continuously as we keep developing
+        new features.
+      </p>
+      <hr>
+    </div>
+    <img class="sponsors-img1 img-fluid" src="{% static 'about/history/UVA-parents-fund-logo.png' %}"/>
+    <img class="sponsors-img2 img-fluid" src="{% static 'about/history/UVA-student-council-logo.png' %}"/>
```

Comment:
Is the idea to make a new class for every new sponsor...? Might not be great for future maintenance, but we probably won't have that many sponsors ever so it's not a big deal.

Reply:
It has to do with the grid positioning
there might be a cleaner way to do it but idk